code-challenges.test.js

```diff
@@ -26,6 +26,16 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("div3", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+        expect(div3(object1.number)).toEqual(true)
+        expect(div3(object2.number)).toEqual(true)
+        expect(div3(object3.number)).toEqual(false)
```

Comment:
The prompt says the function should take an object as an argument. This test is giving it a number. The function should also be returning a string. You are telling the test to expect a Boolean.